README.md

```diff
@@ -8,12 +8,13 @@ If you are not familiar with React library read [react tutorial](http://facebook
 
 To integrate in Dart project add dependency [react](https://pub.dartlang.org/packages/react) to pubspec.yaml.
 
-Include native react library (provided with this library for compatibility reasons) to index.html and create element where you'll mount the react component you'll create.
+Include the native javascript `react` and `react_dom` libraries (provided with this library for compatibility reasons) in `index.html` and create element where you'll mount the react component you'll create.
 
 ```html
 <html>
   <head>
     <script async src="packages/react/react.js"></script>
+    <script async src="packages/react/react_dom.js"></script>
```

Comment:
@maxwellpeterson-wf Good idea, do you think that should be provided for both the development and minified versions, or just the minified version?